eth_keyfile/keyfile.py

```diff
@@ -84,32 +128,37 @@ def normalize_keys(keyfile_json):
 # Version 3 creators
 #
 DKLEN = 32
-SCRYPT_R = 1
-SCRYPT_P = 8
+SCRYPT_R = 8
```

Comment:
I'm not a fan of changing this in case others have used version 3 to encode. I'd rather add a version 4 with the correction and there was a possible addition of more information we could group there as well. I have that locally. Might be best as a separate PR as well. I can submit that if we decide to head that route.

Reply:
I considered that, but decided to add here since it's a due to be released with a major (actually minor since it's only at 0.6.1). The pre-fix v3 would continue to be available in versions <0.7.0.

Reply:
We can then do a separate PR for v4, which includes different fields. Could most likely do something similar to `eth-account`, comparing our results to encoding/decoding from javascript or go implementations.

Reply:
For reference: 
v3 - https://github.com/ethereum/wiki/wiki/Web3-Secret-Storage-Definition
v4 - https://eips.ethereum.org/EIPS/eip-2335

Reply:
If we add some good documentation around this change I _might_ be ok with that. I still don't love it. If I encode something with a `v3` I expect `v3` to decode it. That's kind of the point of versioning imo. I can imagine this causing unnecessary panic on the user side.

Reply:
Well, yeah with them not just being "recommended values" and baked into those definitions, that's not really arguable that it's implemented wrong. Yeah let's stick to the bugfix @pacrob

Reply:
@pacrob did you want to add documentation around the changes here or separately?

Reply:
I will add notes to the readme docs in this PR, I just want to run some comparisons and possibly provide examples. I think comparison testing (if we go there) can wait for v4.